pkg/ui/chooser/embed/index.html

```diff
@@ -16,11 +16,21 @@
             grid-template-columns: 1fr 1fr;
             column-gap: 2rem;
         }
+
+        .logo {
+            display: block;
+            width: 250px;
+            height: auto;
+        }
     </style>
+
+    <link rel="icon" type="image/x-icon" href="../../../../website/static/favicon.ico">
 </head>
 
 <body>
     <header>
+        <img class="logo" src="../../../../website/static/logo/project-logo-dark.svg" alt="full logo">
```

Comment:
the theme we use can adapt to both dark and light mode — could you please ensure the right icon gets used for the right mode?

Reply:
Presently, the chooser ui does not have a light mode

Reply:
oh it totally does!
Dark:
![image](https://github.com/G-Research/fasttrackml/assets/3692455/f356f1ad-7ca6-4ebf-b49c-4ad6f86d0b55)
Light:
![image](https://github.com/G-Research/fasttrackml/assets/3692455/1c58cd1a-72c5-48d3-9bce-62ab3f3ca003)

Reply:
Interesting. I can’t seem to figure this out on my own end. There is no visible toggle. How are you changing it?

Reply:
by changing the mode at the OS level